Resources/Prototypes/_SCP/Entities/Clothing/Uniforms/jumpsuits.yml

```diff
@@ -0,0 +1,141 @@
+  components:
+  - type: Clothing
+    clothingVisuals:
+      jumpsuit:
+      - state: equipped-INNERCLOTHING
+      - state: dept
+        color: "#bb9042"
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformClassD
+  name: class-d jumpsuit
+  description: A standard-issue orange jumpsuit, ideal for a class-d personnel working in a controlled environment.
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/classd.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/classd.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformSecurity
+  name: guard uniform
+  description: A sterile white uniform. Currently issued to  guard personnel.
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/lczguard.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/lczguard.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformSecurityLCZSeargent
+  name: lcz seargent uniform
+  description: A sterile white uniform. Currently issued to LCZ seargents.
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/lczseargent.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/lczseargent.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformSecurityCommander
+  name: director of security's uniform
+  description: A white tactical shirt, with a pair of black trousers with golden striping on the side, the shirt is covered in gold insignia, with an additional black color over the wrists. There's a golden badge and belt buckle. This is definitely the definition of prospertiy.
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/guardcom.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/guardcom.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformSecurityAlt
+  name: guard uniform
+  description: A sterile black uniform. Currently issued to some guard personnel.
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/hczguard.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/hczguard.rsi
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformEngineeringDirectorJumpsuit
+  name: chief containment engineer's jumpsuit
+  description: It's a high visibility jumpsuit given to those engineers insane enough to achieve the rank of \"Director of Maintenance\". It has minor radiation shielding.
+  components:
+  - type: Sprite
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/engineeringdirector.rsi
+  - type: Clothing
+    sprite: _SCP/Clothing/Uniforms/Jumpsuit/engineeringdirector.rsi
+
+
+- type: entity
+  parent: ClothingUniformBase
+  id: SCPClothingUniformUtility
+  name: utility coveralls
```

Comment:
remove

Reply:
It's the parent and the base for the department uniforms, it also works as an unmarked coveralls for other personnel.